From c420bc598ced479cee663477bafaa30ab932ec96 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Thu, 18 Dec 2014 11:59:05 -0500
Subject: [redhat] kabi: Adding symbol bitmap_release_region (lib/bitmap.c)

Message-id: <dac1c186a2924ce2fe840457100cbc24e2c67e6e.1418903403.git.poros@redhat.com>
Patchwork-id: 102240
O-Subject: [PATCH RHEL7.1 13/48] kabi: Adding symbol bitmap_release_region (lib/bitmap.c)
Bugzilla: 1160762
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

Bugzilla: 1160762
    https://bugzilla.redhat.com/show_bug.cgi?id=1160762

    Description:
    ============
    Used in cleanup if failed to get irq for PCI-RT device.
    Used in pcie_rt.ko

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/bitmap_release_region
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64le/bitmap_release_region
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/bitmap_release_region
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/bitmap_release_region

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/bitmap_release_region b/redhat/kabi/kabi-module/kabi_ppc64/bitmap_release_region
new file mode 100644
index 0000000..f823804
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/bitmap_release_region
@@ -0,0 +1,2 @@
+#1-
+0x8251bcc3 bitmap_release_region vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_ppc64le/bitmap_release_region b/redhat/kabi/kabi-module/kabi_ppc64le/bitmap_release_region
new file mode 100644
index 0000000..f823804
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64le/bitmap_release_region
@@ -0,0 +1,2 @@
+#1-
+0x8251bcc3 bitmap_release_region vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/bitmap_release_region b/redhat/kabi/kabi-module/kabi_s390x/bitmap_release_region
new file mode 100644
index 0000000..f823804
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/bitmap_release_region
@@ -0,0 +1,2 @@
+#1-
+0x8251bcc3 bitmap_release_region vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/bitmap_release_region b/redhat/kabi/kabi-module/kabi_x86_64/bitmap_release_region
new file mode 100644
index 0000000..f823804
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/bitmap_release_region
@@ -0,0 +1,2 @@
+#1-
+0x8251bcc3 bitmap_release_region vmlinux EXPORT_SYMBOL
-- 
1.7.1